__tests__/app.test.js

```diff
@@ -47,6 +47,80 @@ describe("/api/articles", () => {
         });
       });
   });
+  it("GET - 200: Should return only those with given topic.", () => {
```

Comment:
Add another test for a topic that has no articles - should return an empty array.